projects/bp-gallery/src/components/common/IfFlagEnabled.tsx

```diff
@@ -0,0 +1,13 @@
+import { IfFeatureEnabled } from '@growthbook/growthbook-react';
+import { PropsWithChildren } from 'react';
+import { FeatureId } from '../../helpers/growthbook';
+
+const growthbookApiHost = import.meta.env.VITE_REACT_APP_GROWTHBOOK_APIHOST;
+const growthbookClientKey = import.meta.env.VITE_REACT_APP_GROWTHBOOK_CLIENTKEY;
+
+export const IfFlagEnabled = ({ children, feature }: PropsWithChildren<{ feature: FeatureId }>) => {
+  if (growthbookApiHost && growthbookClientKey) {
```

Comment:
maybe export `growthbook` from `GrowthBookProvider.tsx` and check for it not being `undefined` instead? avoids a little repetition :)